charts/bulk-scan-processor/values.yaml

```diff
@@ -16,7 +16,6 @@ java:
     REUPLOAD_DELAY: "4000"
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_ENABLED: "true"
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
-    STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
```

Comment:
I think we still need this in `values.yaml` as this will be deployed to actual services and application config doesn't have the knowledge which containers to process hence the processor will fail